src/modules/test/recipes/TpmTests.json

```diff
@@ -5,5 +5,40 @@
         "Desired": false,
         "ExpectedResult": 0,
         "WaitSeconds": 0
+    },
+    {
+        "ComponentName": "Tpm",
```

Comment:
Please use 2 spaces per tab for JSON. Not 4. See how the MIM JSON is for example.